README.md

```diff
@@ -51,17 +55,329 @@ npm i @xmtp/react-native-sdk
+You can get a list of all conversations that have one or more messages.
+
+```tsx
+const allConversations = await xmtp.conversations.list()
+// Say gm to everyone you've been chatting with
+for (const conversation of allConversations) {
+  console.log(`Saying GM to ${conversation.peerAddress}`)
+  await conversation.send('gm')
+}
+```
+
+These conversations include all conversations for a user **regardless of which app created the conversation.** This functionality provides the concept of an [interoperable inbox](https://xmtp.org/docs/concepts/interoperable-inbox), which enables a user to access all of their conversations in any app built with XMTP.
+
+### Listen for new conversations
+
+You can also listen for new conversations being started in real-time. This will allow applications to display incoming messages from new contacts.
+
+**Warning**: This stream will continue infinitely. To end the stream you can either break from the loop, or call `await stream.return()`.
+
+```tsx
+const stream = await xmtp.conversations.stream()
+for await (const conversation of stream) {
+  console.log(`New conversation started with ${conversation.peerAddress}`)
+  // Say hello to your new friend
+  await conversation.send('Hi there!')
+  // Break from the loop to stop listening
+  break
+}
+```
+
+### Start a new conversation
+
+You can create a new conversation with any Ethereum address on the XMTP network.
+
+```tsx
+const newConversation = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897'
+)
+```
+
+## Handle messages
+
+To be able to send a message, the recipient must have already started their client at least once and consequently advertised their key bundle on the network. Messages are addressed using wallet addresses. The message payload can be a plain string, but other types of content can be supported through the use of `SendOptions` (see [Different types of content](#different-types-of-content) for more details)
+
+```tsx
+const conversation = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897'
+)
+await conversation.send('Hello world')
+```
+
+### List messages in a conversation
+
+You can receive the complete message history in a conversation by calling `conversation.messages()`
+
+```tsx
+for (const conversation of await xmtp.conversations.list()) {
+  // All parameters are optional and can be omitted
+  const opts = {
+    // Only show messages from last 24 hours
+    startTime: new Date(new Date().setDate(new Date().getDate() - 1)),
+    endTime: new Date(),
+  }
+  const messagesInConversation = await conversation.messages(opts)
+}
+```
+
+### List messages in a conversation with pagination
+
+It may be helpful to retrieve and process the messages in a conversation page by page.
+
+```tsx
+const conversation = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897'
+)
+
+for await (const page of conversation.messages({ limit: 25 })) {
```

Comment:
React question @daria-github can you pass things as an object? Even if it is setup to handle items. So I'd expect this to be actually `conversation.messages(limit: 25)` but wasn't sure if the object was hurting anything?

Reply:
you can definitely pass an object in general — e.g. if this method was expecting an options object with a limit key this would be the correct syntax —  but looking at this function specifically it seems to just take in   individual arguments where we'd want to say` conversation.messages(25) `instead.